### PR TITLE
Form validation

### PR DESCRIPTION
**Pull request by benshidanny11**
- Implemented form validation email input
- Created styles for a paragraph to display error message

### CODE REVIEW

File: fromvalidation.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+const contactMe = document.getElementById('contactme-form');
+
+function isLowerCase(str) {
+  return !/[A-Z]/.test(str);
+}
```
Comment (SajjadAhmad14) [inline fromvalidation.js:5]: Please use linter from [this](https://github.com/microverseinc/linters-config/tree/master/html-css-js) config which is for both JavaScript and Html&CSS.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Daniel Urimubenshi, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
- [ ] GitHub flow is followed.
- [ ] README is professional.
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by barackm — CHANGES_REQUESTED** [review]
Hi @benshidanny11, 👋 

You have done an amazing job so far! 👏👏👏
However, there are some issues that you still need to work on to go to the next project. But remember you are almost there 😉  😉 !

# STATUS: Required Changes ♻️
### _Check the comments under the review._ 👇 

- [ ] Kindly let's use the linters for JS, HTML & CSS as requested by the previous reviewer, because for now in the `.github/workflows/linters.yml` there is no configuration for javascript, with this `GitHub actions` is not going to check your javascript files. let's use [this file](https://github.com/microverseinc/linters-config/blob/master/html-css-js/.github/workflows/linters.yml) instead 👍 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by safafa** [comment]
## STATUS: APPROVED 🟢 👏
Hello, @benshidanny11  great job working on the required changes 👍 💯

Now you can merge this branch 💪 😄
HAPPY CODING!! AND GOOD LUCK WITH THE NEXT ONES 🤞 🍀

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by safafa — APPROVED** [review]
## STATUS: APPROVED 🟢 👏
Hello, @benshidanny11  great job working on the required changes 👍 💯

Now you can merge this branch 💪 😄
HAPPY CODING!! AND GOOD LUCK WITH THE NEXT ONES 🤞 🍀

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**